content/source/docs/extend/index.html.md

```diff
@@ -1,29 +1,27 @@
 ---
 layout: "extend"
-page_title: "Home - Extending Terraform"
-sidebar_current: "docs-extend-home"
+page_title: "Home - Building Terraform Providers"
 description: |-
-  Extending Terraform is a section for content dedicated to developing Plugins
-  to extend Terraform's core offering.
+  Building Terraform Providersis a section for content dedicated to developing
+  providers to extend Terraform's core offering.
 ---
 
-# Extending Terraform
+# Building Terraform Providers
 
 Terraform can be extended to allow users to manage more infrastructure providers with
```

Comment:
Since "providers" is immediately afterwards and being lowercased, might make sense to drop the word here or as suggested reword this a little more.

```suggestion
Terraform can be extended to manage infrastructure with
```